portfolio/src/main/webapp/script.js

```diff
@@ -79,7 +80,8 @@ function createMap() {
 }
 
 function addImageToImagesContainer(data) {
-  $('#images-container').append(`<img src=${data.imageUrl}><img>`);
+  $('.image-presenter').empty();
+  $('.image-presenter').append(`<img src=${data.imageUrl}><img>`);
```

Comment:
FYI I'm pretty sure you can do this instead of empty/append:

```javascript
$('.image-presenter').html(`<img src=${data.imageUrl}><img>`);
```

Reply:
awesome! done